backend/app/services/employee_emergency_contact_service.py

```
@@ -148,15 +151,16 @@ async def create_emergency_contact(
         logger.error(f"Authorization error: {str(e)}", extra={"request_id": request_id})
         raise
     except Exception as e:
-        logger.error(f"Unexpected error creating emergency contact for user_id {current_user.user_id}: {str(e)}", extra={"request_id": request_id})
+        logger.error(f"Unexpected error creating emergency contact for user_id {user_id}: {str(e)}", extra={"request_id": request_id, "user_id": user_id})
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)